e2e-tests/functions

```diff
@@ -26,16 +26,6 @@ namespace="${test_name}-${RANDOM}"
 replica_namespace="${test_name}-replica-${RANDOM}"
 conf_dir=$(realpath $test_dir/../conf || :)
 src_dir=$(realpath $test_dir/../..)
-logs_dir=$(realpath $test_dir/../logs)
```

Comment:
@pooknull, why did you remove it?

Reply:
I thought it was unused: https://github.com/percona/percona-xtradb-cluster-operator/pull/1743/commits/2e985f847ae2104ade193dad50f4bb9236b73c03